com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDAssetFactory.cs

```diff
@@ -59,7 +59,7 @@ public override void Action(int instanceId, string pathName, string resourceFile
         }
 
         // Hide: User aren't suppose to have to create it.
-        //[MenuItem("Assets/Create/Rendering/High Definition Render Pipeline Resources", priority = CoreUtils.assetCreateMenuPriority1)]
+        //[MenuItem("Assets/Create/Rendering/HDRP Resources", priority = CoreUtils.Sections.k_Section7 + CoreUtils.Priorities.k_AssetsCreateRenderingMenuPriority)]
```

Comment:
I know this is a comment but uncommenting it will not work. Renaming failed of k_Section7. (Same for several other commented)